agent/src/test/java/com/microsoft/applicationinsights/agent/internal/config/XmlAgentConfigurationBuilderTest.java

```diff
@@ -75,6 +76,16 @@ public void testBuiltInConfiguration() throws IOException {
         assertEquals(builtInConfiguration.isHibernateEnabled(), false);
     }
 
+    @Test
+    public void testLoggingConfiguration() throws IOException {
+        AgentConfiguration configuration = testConfiguration("LoggingTest.xml");
+        Map<String, String> loggingConfig = configuration.getAgentLoggingConfiguration();
+        assertNotNull(loggingConfig);
+        assertEquals("TRACE", loggingConfig.get("Level"));
```

Comment:
It's good practice to pull information from the map using the same method as the source implementation (use the constants in case they change).

Reply:
This is indeed true. I will update this one.

Reply:
Done